backend/src/EntityExtractor.Web/Models/UploadFileRequestModel.cs

```diff
@@ -0,0 +1,4 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace eKultura.EntityExtractor.Web.Models;
+public record UploadFileRequestModel(IFormFile File, string Topic);
```

Comment:
Generally, Model suffix is used for MVC architecture. I would suggest naming it only `UploadFileRequest`